universal_data_permissions_scanner/datastores/snowflake/analyzer.py

```diff
@@ -92,10 +96,14 @@ def connect(
         # Handle case sensitive warehouse name, wrap with quotes
         warehouse = f'"{warehouse}"'
 
+        if rsa_key is not None:
```

Comment:
Why do we pass some in kwargs an other as field ?

Reply:
Do you see a problem with that?

Reply:
It's weird, like what is the decision point?